unleash-yggdrasil/src/strategy_parsing.rs

```diff
@@ -371,32 +380,23 @@ fn rollout_constraint(node: Pairs<Rule>) -> CompileResult<RuleFragment> {
         .transpose()?;
 
     Ok(Box::new(move |context: &Context| {
-        let stickiness = stickiness_resolver(context);
+        if let Some(stickiness) = stickiness_resolver(context) {
 
-        if stickiness.is_none() {
-            return false;
-        }
-
-        let group_id = match &group_id {
-            Some(group_id) => group_id.clone(),
-            None => context.toggle_name.clone(),
-        };
+            let group_id = match &group_id {
+                Some(group_id) => group_id.clone(),
+                None => context.toggle_name.clone(),
+            };
 
-        let hash = if let Some(stickiness) = stickiness {
-            normalized_hash(&group_id, &stickiness, 100, 0)
-        } else {
-            // The original code does something different here - if we're using the
-            // default strategy it generates a string of a number between 1 and 101
-            // then uses that as the hash. This instead doesn't do that and just
-            // uses a random number in place of the hash. Pretty sure it's the same thing
-            Ok(rand::thread_rng().gen_range(0..99) as u32)
-        };
```

Comment:
so this whole thing goes away now, yea? I take it that means that if you use random or sticky on a property that doesn't exist in the context, you get "random" stickiness instead?

Reply:
We spoke but for anyone arriving later - turns out this code was a dead branch and this is just pruning now